### PR TITLE
Add Makerlog-menubar.app v1.0

### PR DESCRIPTION
**Pull request by clawfire**
After making all changes to the cask:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] The commit message includes the cask’s name and version.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, if **adding a new cask**:

- [x] Named the cask according to the [token reference].
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked there are no [open pull requests] for the same cask.
- [x] Checked the cask was not [already refused].
- [x] Checked the cask is submitted to [the correct repo].

[token reference]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md
[open pull requests]: https://github.com/Homebrew/homebrew-cask/pulls
[already refused]: https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues
[the correct repo]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask
[version-checksum]: https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/stanzas/sha256.md#updating-the-sha256

### CODE REVIEW

File: Casks/makerlog-menubar.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+  version '1.0.0'
+  sha256 'a222d3ad5aee74fa2c7ed5892eeada373223d3eedbac7d82e5647fb5aa5d97b2'
+
+  # github.com/Booligoosh/makerlog-menubar/ was verified as official when first introduced to the cask
+  url 'https://github.com/Booligoosh/makerlog-menubar/releases/download/v1.0.0/Makerlog-Menubar-for-Mac-v1.0.0.zip'
+  appcast 'https://github.com/Booligoosh/makerlog-menubar/releases.atom'
+  name 'Makerlog Menubar'
+  homepage 'https://menubar.getmakerlog.com/'
+
+  app ''
```
Comment (vitorgalvao) [inline Casks/makerlog-menubar.rb]: You forgot this.

File: Casks/makerlog-menubar.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
+cask 'makerlog-menubar' do
+  version '1.0.0'
+  sha256 'a222d3ad5aee74fa2c7ed5892eeada373223d3eedbac7d82e5647fb5aa5d97b2'
+
+  # github.com/Booligoosh/makerlog-menubar/ was verified as official when first introduced to the cask
+  url 'https://github.com/Booligoosh/makerlog-menubar/releases/download/v1.0.0/Makerlog-Menubar-for-Mac-v1.0.0.zip'
```
Comment (vitorgalvao) [inline Casks/makerlog-menubar.rb]: This should use string interpolation.

**Comment by vitorgalvao** [comment]
Thank you, but we’ll be rejecting this on account of [not being notable enough](https://github.com/Homebrew/homebrew-cask/blob/master/doc/faq/rejected_casks.md):

> The Cask is too obscure (for example, a self-submitted app from a GitHub repository that is [not notable enough](https://github.com/Homebrew/homebrew-cask/pull/28103) (under 50 stars)).

**Comment by clawfire** [comment]
Didn't know about the rule of not being notable enough :O but makes sense. Thanks for your comments @vitorgalvao and the time you take to review my PR.

**Pull request closed without merging**